package.json

```diff
@@ -791,6 +796,38 @@
           "group": "new2@40",
           "when": "view == javaProjectExplorer && (viewItem =~ /java:(file|folder|project)/ || viewItem =~ /java:(packageRoot)(?=.*?\\b\\+resource\\b)/)"
         }
+      ],
+      "javaProject.newJavaFile": [
+        {
+          "command": "java.view.package.newJavaClass",
+          "group": "new@10",
+          "when": "explorerResourceIsFolder"
```

Comment:
Any reason that we do not take file node into consideration?

Reply:
this is same as the behavior of the standard VS Code "New File/Folder ..." menu, which only appears on the folder context.

Reply:
ok